src/NServiceBus.AcceptanceTests/Mutators/Issue_1980.cs

```diff
@@ -8,18 +8,17 @@
 
     public class Issue_1980 : NServiceBusAcceptanceTest
     {
+        static Context testContext = new Context();
```

Comment:
Let me rephrase. It's not that it doesn't work. I made it much harder so that you don't forget to await the result of the scenario run.

About the static. Consistency is the key. Now we have 3 tests which do it different...